---a/net/minecraft/item/ItemSpade.java
+++b/net/minecraft/item/ItemSpade.java
@@ -1,22 +1,21 @@
 package net.minecraft.item;
 
-import com.google.common.collect.Sets;
 import java.util.Set;
+
 import net.minecraft.block.Block;
 import net.minecraft.init.Blocks;
 
-public class ItemSpade extends ItemTool
-{
-    private static final Set EFFECTIVE_ON = Sets.newHashSet(new Block[] {Blocks.clay, Blocks.dirt, Blocks.farmland, Blocks.grass, Blocks.gravel, Blocks.mycelium, Blocks.sand, Blocks.snow, Blocks.snow_layer, Blocks.soul_sand});
+import com.google.common.collect.Sets;
+
+public class ItemSpade extends ItemTool {
+
+    private static final Set EFFECTIVE_ON = Sets.newHashSet(new Block[]{Blocks.clay, Blocks.dirt, Blocks.farmland, Blocks.grass, Blocks.gravel, Blocks.mycelium, Blocks.sand, Blocks.snow, Blocks.snow_layer, Blocks.soul_sand});
     private static final String __OBFID = "CL_00000063";
 
-    public ItemSpade(Item.ToolMaterial material)
-    {
+    public ItemSpade(Item.ToolMaterial material) {
         super(1.0F, material, EFFECTIVE_ON);
     }
 
-    public boolean canHarvestBlock(Block blockIn)
-    {
+    public boolean canHarvestBlock(Block blockIn) {
         return blockIn == Blocks.snow_layer ? true : blockIn == Blocks.snow;
     }
-}
+}
